### PR TITLE
RestartNextRound when Exiled update

### PR DESCRIPTION
**Pull request by louis1706**
wait for to be merged #1941

### CODE REVIEW

**Comment by louis1706** [comment]
Would like someone to test it on other server than windows

File: Exiled.Updater/Updater.cs [hunk 1]
```diff
@@ -261,88 +292,94 @@ where name.StartsWith("Exiled.", StringComparison.OrdinalIgnoreCase)
...
         #endregion
 
-        private void Update(HttpClient client, NewVersion newVersion)
+        private async Task Update(HttpClient client, NewVersion newVersion)
         {
             try
             {
                 Log.Info("Downloading installer...");
-                using (HttpResponseMessage installer = client.GetAsync(newVersion.Asset.BrowserDownloadUrl).ConfigureAwait(false).GetAwaiter().GetResult())
+                HttpResponseMessage installerResponse = client.GetAsync(newVersion.Asset.BrowserDownloadUrl).ConfigureAwait(false).GetAwaiter().GetResult();
```
Comment (louis1706) [inline Exiled.Updater/Updater.cs:300]: I didn't find how to make this one await without error

Comment (joker-119) [reply Exiled.Updater/Updater.cs:301]: ok but HttpResponseMessage is an IDisposable so where's my using directive so it's disposed properly?

And what was the error when using a normal await?

**Comment by louis1706** [comment]
Let's rewrite this PR from 0 and doing it better

**Pull request closed without merging**